### PR TITLE
[test-operator][tobiko] Change downloaded images path

### PR DESCRIPTION
**Pull request by lpiwowar**
This patch changes the default path for downloaded images used for
tobiko testing from ~/.downloaded-images/ubuntu-minimal to
/usr/local/share.

This change is needed because we have to free /var/lib/tobiko from any
files that are present prior the container execution. This allows us to
mount ephemeral-storage to /var/lib/tobiko and use this directory as the
only point where write actions are allowed when readOnlyRootFilesystem:
true.

### CODE REVIEW

**Comment by lpiwowar** [comment]
/hold

This PR requires review from: @eduolivares

**Comment by eduolivares** [comment]
looks good to me. I would just add "Depends-On: https://github.com/openstack-k8s-operators/tcib/pull/233"

**Review by slawqo — APPROVED** [review]

**Comment by lpiwowar** [comment]
@eduolivares there is cyclic dependency between those two PRs. We will have to schedule both PRs to be merged ideally at the same time.

**Review by eduolivares — APPROVED** [review]

**Review by frenzyfriday — APPROVED** [review]

**Pull request merged**
